src/useLastLocation.spec.tsx

```diff
@@ -1,137 +1,68 @@
+/* eslint-disable import/no-extraneous-dependencies */
 import * as React from 'react';
-import { mount, ReactWrapper } from 'enzyme';
-import { MemoryRouter, Route, Link } from 'react-router-dom';
+import { renderHook as rtlRenderHook, act } from '@testing-library/react-hooks';
+import { Router } from 'react-router-dom';
```

Comment:
Given the test is using `react-router-dom`, I wonder if it's better to use `@testing-library/react-hooks/dom` and remove any ambiguity about which renderer and what sort of environments is being used?

Reply:
I agree.